src/components/CardTable/CardTable.jsx

```diff
@@ -109,14 +109,16 @@ export default class CardTable extends Component {
   }
 
   get largestTableWidth () {
-    const tableNodes = Object.values(this.tableNodes)
+    const tableNodes = Object.values(this.tableNodes || {})
     const tableWidths = tableNodes.map(tableNode => tableNode.clientWidth)
 
     return Math.max(...tableWidths)
   }
 
   get parentWidth () {
-    return getElementContentWidth(this.containerNode.parentNode)
+    const ZERO = 0
+
+    return this.containerNode ? getElementContentWidth(this.containerNode.parentNode) : ZERO
```

Comment:
On initial render it seems to call the function twice even when bound. While the react docs say you should bind it to stop it being called twice on renders I'm not sure this is possible on the initial render. Also instantiating the container node within the class won't do anything as it's being overwritten as null before we touch it. Although it is happening within the promise, so it could be an issue stemming from the asynchronicity. 

Regardless, after looking at a few other libraries for solutions, I think it is much easier to save any potential future headache and just do a simple truthy check. It seems to be very common practice.